repo.go

```diff
@@ -526,6 +534,50 @@ func (r *Repo) Sign(name string) error {
 	return r.local.SetMeta(name, b)
 }
 
+// AppendSignature allows users to append a signature generated with an external tool.
+func (r *Repo) AppendSignature(name string, signature data.Signature) error {
```

Comment:
What is `name` exactly? Only the base, role filename?

Reply:
Manifest name! Clarified in the comment. name typically referred to that in this file, although I agree it's not clear...

Reply:
What do you think about "roleFilename"?

Reply:
Something like that sounds fine! I'll make the change consistent though through the file